Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:latest
```

Comment:
The Dockerfile can be as simple as:

```
FROM scratch
COPY ./dist/linux-amd64/esevaluator /app
EXPOSE 8080
ENTRYPOINT ["/app"]
```

This relies on an up-to-date distribution build and adds an `ENTRYPOINT` so a container can be run with the program as the entrypoint (yup). This allows for:

```
docker run -v ./tls:/tls -p 8080:8080 -d harvest-platform/esevaluator -tlscert /tls/cert.pem -tlskey /tls/key.pem
```